things/postgres/things.go

```diff
@@ -22,6 +22,14 @@ const (
 	errTruncation = "string_data_right_truncation"
 )
 
+var (
+	errSaveDB     = errors.New("Save to DB failed")
+	errUpdateDB   = errors.New("Update to DB failed")
+	errRetrieveDB = errors.New("Retreiving from DB failed")
+	errMarshal    = errors.New("Failed to marshal metadata")
+	errUnmarshal  = errors.New("Failed to unmarshal metadata")
```

Comment:
Use lower case for error messages. You can also improve the consistency starting messages by `failed to ...`